etl/steps/data/meadow/un_sdg/2022-05-26/un_sdg.py

```diff
@@ -120,6 +127,11 @@ def create_dataframe(original_df: pd.DataFrame) -> None:
     output_tables = []
 
     for group_name, df_group in all_series:
+        log.info(
```

Comment:
I've added a bit of logging to see where we are and to help me identify problems if it fails.